src/KubeOps/Operator/Entities/Extensions/KubernetesObjectExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using k8s;
+using k8s.Models;
+using Org.BouncyCastle.Asn1.Tsp;
```

Comment:
Why do you need bouncy castle?

Reply:
uhhhhh.. thats interesting! I suspect VS auto added the namespace and I didn't notice!